ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -127,8 +129,11 @@ public AmbryRequestsTest() throws IOException, ReplicationException, StoreExcept
     dataNodeId = clusterMap.getDataNodeIds().get(0);
     replicationManager =
         MockReplicationManager.getReplicationManager(verifiableProperties, storageManager, clusterMap, dataNodeId);
+    StoreKeyConverterFactory storeKeyConverterFactory =
```

Comment:
will need a mock to check functionality. I think you can write a mock in in ambry-api package (in the tests module). Something that will let you add and remove mappings from the converter either as individual mappings or an input hashmap.

Reply:
Implements `StoreKeyConverterFactory/StoreKeyConverter` in ambry-api  is a little difficulty. They need the context in the test. 

I added them in `AmbryRequestsTest.java`, looks OK.